admin/app/components/solidus_admin/ui/table/component.js

```diff
@@ -83,6 +83,18 @@ export default class extends Controller {
     this.render()
   }
 
+  toggleRowSelection(event) {
+    if (event.target.closest('a') || event.target.tagName === 'BUTTON' || event.target.type === 'checkbox') return
```

Comment:
Added a check for `button` in `toggleRowSelection` to anticipate potential buttons in future user-generated tables.